VENDOR_BRANCHING.md

```diff
@@ -7,6 +7,7 @@ Examples:
 - geonames data (https://www.geonames.org/recent-changes.html), https://github.com/Stellarium/stellarium-data/tags
 - quasar data (https://github.com/Stellarium/stellarium/blob/master/plugins/Quasars/util/quasars.tsv)
 - Almagest data (minor fixes, of course - this is essentially frozen data)
+- HTC algorithms (Helene, Telesto, and Calypso (Lagrangian satellites of Dione) - taken from ftp://ftp.imcce.fr/pub/ephem/satel/htc20/htc20.f ? )
```

Comment:
Only in the unlikely case that we directly add their unmodified Fortran code to the project...

Reply:
that's called Murphy's Law :-)

(and also, when one never tries, one will never become proficient in the pattern, so better try now than never)